gulp/tasks/test.js

```diff
@@ -53,9 +53,15 @@ function testUnitScripts( cb ) {
     fileTestRegex += '.*-spec.js';
   }
 
+  /*
+    The --no-cache flag is needed so the transforms don't cache.
+    If they are cached, preprocessor-handlebars.js can't find handlebars. See
+    https://facebook.github.io/jest/docs/en/troubleshooting.html#caching-issues
```

Comment:
Alternatively to this we could install `handlebars` in the main `package.json`. However, that means two installs of handlebars (in the main package.json and in the oah package.json). I don't see a slow-down without the cache, but ideally we'd address this with `getCacheKey`.